cluster/plan.go

```diff
@@ -451,6 +459,7 @@ func (c *Cluster) BuildKubeletProcess(host *hosts.Host, serviceOptions v3.Kubern
 		"pod-infra-container-image": kubelet.InfraContainerImage,
 		"root-dir":                  path.Join(host.PrefixPath, "/var/lib/kubelet"),
 	}
+	CommandArrayArgs := map[string][]string{}
```

Comment:
Can we do something similar here as with my previous suggestions to save on unnecessary allocations?